.github/workflows/test.yml

```diff
@@ -1,7 +1,8 @@
-# Tests / Code Coverage workflow
-# This workflow is triggered by ci_release.yml workflow or manually
+---
+# This workflow is triggered by ci.yml workflow or manually
 name: Tests / Code Coverage
-on:
+permissions: {}
+"on":
   workflow_call:
     inputs:
       image-tag:
```

Comment:
since we split out the docker workflows, image-tag is no longer required as an argument in test.yml as far as I can tell.